packages/formation/js/accordion.js

```diff
@@ -34,39 +34,44 @@ const loadAccordionHandler = () => {
     .forEach(element => {
       element.addEventListener('click', e => {
         const target = e.target;
-        const other = Array.from(
-          element.getElementsByClassName('usa-accordion-button'),
-        ).filter(item => item !== target);
 
-        const multiSelectable = toBoolean(
-          element.getAttribute('aria-multiselectable'),
-        );
+        // Checks whether the button has a click event already assigned to it.
+        // Specifically React Components.
+        if (!target.onclick) {
+          const other = Array.from(
+            element.getElementsByClassName('usa-accordion-button'),
+          ).filter(item => item !== target);
 
-        if (target.getAttribute('aria-controls')) {
-          if (!multiSelectable) {
-            other.forEach(el => {
-              const contentEl = el.getAttribute('aria-controls');
+          const multiSelectable = toBoolean(
+            element.getAttribute('aria-multiselectable'),
+          );
 
-              el.setAttribute('aria-expanded', 'false');
+          if (target.getAttribute('aria-controls')) {
+            if (!multiSelectable) {
+              other.forEach(el => {
+                const contentEl = el.getAttribute('aria-controls');
 
-              document
-                .getElementById(contentEl)
-                .setAttribute('aria-hidden', 'true');
-            });
-          }
+                el.setAttribute('aria-expanded', 'false');
 
-          const dropDownElement = document.getElementById(
-            target.getAttribute('aria-controls'),
-          );
-          const targetExpandedAttr = toBoolean(
-            target.getAttribute('aria-expanded'),
-          );
+                document
+                  .getElementById(contentEl)
+                  .setAttribute('aria-hidden', 'true');
+              });
+            }
+
+            const dropDownElement = document.getElementById(
+              target.getAttribute('aria-controls'),
+            );
+            const targetExpandedAttr = toBoolean(
```

Comment:
Since this typecast to boolean (and isn't the attribute itself) it might be better named as `hasTargetExpandedAttribute`

Reply:
Actually the problem is that this gives us back a `"true"` value as a string and I need to convert it into a boolean, so I can do the opposite using the `!targetExpandedAttr`. So it actually gives us back true and false but because it is html it gives it back as a string.